zebra/zebra_routemap.c

```diff
@@ -1289,6 +1322,9 @@ static void zebra_route_map_process_update_cb(char *rmap_name)
 	if (IS_ZEBRA_DEBUG_EVENT)
 		zlog_debug("Event handler for route-map: %s",
 			   rmap_name);
+	zebra_import_table_rm_update(rmap_name);
```

Comment:
1) I do not understand on why these 4 lines of code was moved from update_timer function to here. 
2) Don't think this is a right function, as function name and the new code will not be related. Lets discuss